runhouse/resources/hardware/cluster.py

```diff
@@ -2160,10 +2204,14 @@ def install_package(self, package: Union["Package", str]):
             package = package.to(self)
             self.client.install_package(package)
 
-    def install_package_over_ssh(self, package: Union["Package", str], node: str):
+    def install_package_over_ssh(
+        self, package: Union["Package", str], node: str, conda_name: str
+    ):
         from runhouse.resources.packages.package import Package
 
         if isinstance(package, str):
             package = Package.from_string(package)
+            if package.install_method in ["reqs", "local"]:
```

Comment:
you should be able to kill this if statement -- i made it so that `.to` is always fine to do